core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/workspacemanager/HttpWorkspaceManagerDAO.scala

```diff
@@ -199,27 +199,66 @@ class HttpWorkspaceManagerDAO(apiClientProvider: WorkspaceManagerApiClientProvid
     )
   }
 
-  def createAzureStorageContainer(workspaceId: UUID, storageAccountId: Option[UUID], ctx: RawlsRequestContext) = {
+  override def createAzureStorageContainer(workspaceId: UUID,
+                                           storageContainerName: String,
+                                           storageAccountId: Option[UUID],
+                                           ctx: RawlsRequestContext
+  ) = {
     val storageContainerCreationParameters = storageAccountId match {
       case Some(_) =>
         new AzureStorageContainerCreationParameters()
-          .storageContainerName(s"sc-${workspaceId}")
+          .storageContainerName(storageContainerName)
           .storageAccountId(storageAccountId.get)
       case None =>
         new AzureStorageContainerCreationParameters()
-          .storageContainerName(s"sc-${workspaceId}")
+          .storageContainerName(storageContainerName)
     }
 
     getControlledAzureResourceApi(ctx).createAzureStorageContainer(
       new CreateControlledAzureStorageContainerRequestBody()
         .common(
-          createCommonFields(s"sc-${workspaceId}").cloningInstructions(CloningInstructionsEnum.DEFINITION)
+          createCommonFields(storageContainerName).cloningInstructions(CloningInstructionsEnum.DEFINITION)
         )
         .azureStorageContainer(storageContainerCreationParameters),
       workspaceId
     )
   }
 
+  override def cloneAzureStorageContainer(sourceWorkspaceId: UUID,
+                                          destinationWorkspaceId: UUID,
+                                          sourceContainerId: UUID,
+                                          cloningInstructions: CloningInstructionsEnum,
+                                          ctx: RawlsRequestContext
+  ): CloneControlledAzureStorageContainerResult = {
```

Comment:
For methods that are straight passthroughs, I haven't been creating unit test cases in `HttpWorkspaceManagerDAOSpec`. But perhaps that is just lazy? And I suppose this isn't technically a "straight passthrough", so can add test coverage if desired.